### DISCUSSION TITLE
Split channels for spectrogram

### DISCUSSION THREAD

**Discussion by sophie-cal2020** [katspaugh/wavesurfer.js#3037] [Q&A]
Hello,

May I know if it is possible to display split channels for the spectrogram plugin?  I tried setting the splitChannels option in wavesurfer.  It shows 2 waveforms but only 1 spectrogram.  Thanks!

**Comment by katspaugh** [comment]
Looks like this plugin has channels=1 hardcoded, so it will render only the first channel.

**Reply by sophie-cal2020** [reply]
Hi, I noticed that the problem has been solved.  There is now a splitChannel option in the Spectrogram plugin.  Thank you!
